website/docs/r/org_ldap.html.markdown

```diff
@@ -80,12 +80,27 @@ resource "vcd_org_ldap" "my-org-ldap" {
 }
 ```
 
+## Example Usage 2 - Using system configuration
+
+```hcl
+data "vcd_org" "my-org" {
+  name = "my-org"
+}
+
+resource "vcd_org_ldap" "my-org-ldap" {
+  org_id         = data.vcd_org.my-org.id
+  ldap_mode      = "SYSTEM"
+  custom_user_ou = "ou=Foo,dc=domain,dc=local base DN"
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
 * `org_id` - (Required) Org ID: there is only one LDAP configuration available for an organization. Thus, the resource can be identified by the Org.
 * `ldap_mode` - (Required) One of `NONE`, `CUSTOM`, `SYSTEM`. Note that using `NONE` has the effect of removing the LDAP settings
+* `custom_user_ou` - (Optional) If `ldap_mode` is `SYSTEM`, specifies a LDAP `attribute=value` pair to use for OU (organizational unit)
```

Comment:
Nit:

```suggestion
* `custom_user_ou` - (Optional) If `ldap_mode` is `SYSTEM`, specifies an LDAP `attribute=value` pair to use for OU (organizational unit)
```

p.s. "a" vs. "an" depends on the consonant/vowel **sound**:

https://dictionary.cambridge.org/grammar/british-grammar/a-an-and-the

>     /ən empi:θri: …/ an MP3 player

Reply:
Fixed